docs/deployment-notes/puppeteer/configure-page-layout-assignments.mjs

```
@@ -0,0 +1,142 @@
+      timeout
+    });
+
+    // Find the option element with the matching text content
+    const option = await frame.$(`option[value][text="${pageLayoutName}"]`);
+
+    // If the option is found, set the dropdown value directly
+    if (option) {
+      await frame.evaluate(
+        (d, o) => {
+          d.value = o.value;
+          d.dispatchEvent(new Event('change'));
```

ESLint

[error] Prefer using "CustomEvent" constructor over "Event" for dispatching events.

(@lwc/lwc/prefer-custom-event)